I've added a failing test, I'm investigating the fix now, but it's 100% a problem with that test, seeing locally:

```
$ go test ./internal/rbd
--- FAIL: TestValidateImageFeatures (0.00s)
    rbd_util_test.go:134: 
                Error Trace:    rbd_util_test.go:134
                Error:          Expected nil, but got: &errors.errorString{s:"invalid feature "}
                Test:           TestValidateImageFeatures
FAIL
FAIL    github.com/ceph/ceph-csi/internal/rbd   0.048s
FAIL
```

--- a/internal/rbd/rbd_util.go
+++ b/internal/rbd/rbd_util.go
@@ -993,6 +993,13 @@ func genVolFromVolumeOptions(ctx context.Context, volOptions, credentials map[st
 }
 
 func (rv *rbdVolume) validateImageFeatures(imageFeatures string) error {
+	// It is possible for image features to be an empty string which
+	// the Go split function would return a single item array with
+	// an empty string, causing a failure when trying to validate
+	// the features.
+	if strings.TrimSpace(imageFeatures) == "" {
+		return nil
+	}
 	arr := strings.Split(imageFeatures, ",")
 	featureSet := sets.NewString(arr...)
 	for _, f := range arr {

--- a/internal/rbd/rbd_util_test.go
+++ b/internal/rbd/rbd_util_test.go
@@ -52,6 +52,14 @@ func TestValidateImageFeatures(t *testing.T) {
 		errMsg        string
 	}{
 		{
+			"",
+			&rbdVolume{
+				Mounter: rbdDefaultMounter,
+			},
+			false,
+			"",
+		},
+		{
 			"layering",
 			&rbdVolume{
 				Mounter: rbdDefaultMounter,

